deviations/migrations/0004_auto_20211207_1459.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.2.4 on 2021-12-07 12:59
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('userprofile', '0006_auto_20210812_1536'),
+        ('deviations', '0003_auto_20210812_1536'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='deadlineruledeviation',
+            name='grant_time',
+            field=models.DateTimeField(auto_now=True, null=True, verbose_name='LABEL_GRANT_TIME'),
```

Comment:
Granter and grant time will be None for pre-existing deviations (I tested). I think this makes sense because we don't know when the old deviations were created.